src/llmcompressor/transformers/tracing/llava.py

```diff
@@ -58,7 +58,7 @@ def maybe_install_metadata_image_features(
 
 # TRACING: The shape of inputs_embeds is known. This function compensates for
 # the fact that shape inference through `masked_scatter` is not implemented yet
-def maybe_install_metadata_inputs_embeds(
+def maybe_install_metadata_inputs_embeds_masked(
```

Comment:
I think needs rebase?

Reply:
The diff has disappeared since you merged from main